### PR TITLE
docs: add MIT license

### PR DESCRIPTION
**Pull request by brandon-charest**
## Description
regex issue when generating list of plugins.

Added MIT License file.

## Check List

- [x] I have read the [Contributing Guidelines](./CONTRIBUTING.md)

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,24 @@
+MIT License
```
Comment (mcollina) [inline LICENSE:1]: I think it's better that @Eomm add this, he authored the site after all

Comment (Eomm) [reply LICENSE:1]: It is fine 👍🏼

File: scripts/build-plugin-list.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ async function extractEcosystemFromFile(file) {
 
 function extractPlugins(pluginContent) {
   const lines = pluginContent.split('\n').filter(Boolean) // remove empty lines
-
+  console.log(lines)
```
Comment (Eomm) [inline scripts/build-plugin-list.js]: could you remove the changes here?

so this PR can focus on 1 single thing at time

Comment (brandon-charest) [reply scripts/build-plugin-list.js]: done

File: LICENSE (hunk 1 above)
Comment (brandon-charest) [reply LICENSE:1]: np! removed file to keep scope of PR to a single change

**Comment by mcollina** [comment]
There is a conflict now

**Comment by Uzlopak** [comment]
Plugin formatting was already done in #126 and got merged. Thats why @Eomm  changed the title to adding the License, to give @brandon-charest the chance to still provide a PR. But it seems, Brandon did not notice that, so changed back to fixing the formatting, removed the License and changed back the title.

**Comment by brandon-charest** [comment]
I did not notice. I thought that was my mistake haha. I can revert and add the license

**Comment by Uzlopak** [comment]
@brandon-charest 
Please do it.

File: scripts/build-plugin-list.js [hunk 3]
```diff
@@ -71,7 +71,6 @@ function extractPlugins(pluginContent) {
     const name = match[1]
     const url = match[2]
     const description = match[3] ? match[3].trim().replace(/ {2,}/g, ' ') : ''
-
```
Comment (Uzlopak) [inline scripts/build-plugin-list.js]: Can you add back the deleted line, please

Comment (brandon-charest) [reply scripts/build-plugin-list.js]: Done

**Comment by Uzlopak** [comment]
@brandon-charest 
I dont know. Maybe the removal of the newline was correct for the linter. Please fix the linting issue.
Will approve the change.

File: scripts/build-plugin-list.js [hunk 4]
```diff
@@ -71,6 +71,7 @@ function extractPlugins(pluginContent) {
     const name = match[1]
     const url = match[2]
     const description = match[3] ? match[3].trim().replace(/ {2,}/g, ' ') : ''
+
```
Comment (Uzlopak) [inline scripts/build-plugin-list.js]:
```suggestion

```

**Review by Uzlopak — APPROVED** [review]
I removed the spaces so now this PR adds only the MIT license.

Thanks for your work.

**Review by Eomm — APPROVED** [review]
👍🏼

**Pull request merged**